recipe/meta.yaml

```diff
@@ -6,13 +6,13 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/jupyter-server/fps/archive/{{ name }}-{{ version }}.tar.gz
-  sha256: 55f9541fe478d0022ceba8a6920c32733a4dfde8c7f34fc10f92cd0791b7fa0e
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fps_uvicorn-0.0.19.tar.gz
+  sha256: 249556e6920d3a56e11bef42adbd64944f210ddfb8b0208ca3caadc9764f8b0a
 
 build:
   noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install ./plugins/uvicorn/ --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
related to my comment in https://github.com/conda-forge/fps-feedstock/pull/6, I don't really get why you are not only packaging `fps-uvicorn` but `fps` instead here?

Reply:
We are now getting the source from the PyPI package, so this should be fine.

Reply:
right I didn't paid attention to that, thanks @davidbrochart